cypress/e2e/staffMembers.js

```diff
@@ -163,4 +175,148 @@ describe("Staff members", () => {
         });
     },
   );
+
+  it(
+    "should not be able to create staff member with not unique email. TC: SALEOR_3508",
+    { tags: ["@staffMembers", "@stagedOnly"] },
+    () => {
+      const firstName = faker.name.firstName();
+      const emailInvite = TEST_ADMIN_USER.email;
+      activatePlugin({ id: "mirumee.notifications.admin_email" });
+      cy.visit(urlList.staffMembers)
+        .expectSkeletonIsVisible()
+        .get(STAFF_MEMBERS_LIST.inviteStaffMemberButton)
+        .click({ force: true });
+      fillUpUserDetailsWithNotUniqueEmail(firstName, lastName, emailInvite);
```

Comment:
My suggestion is to move the last check `.confirmationErrorMessageShouldAppear();`
from function to test body just to be clear about what was expected in the tests

Reply:
ok
(@michalina-graczyk)